src/styles/components/footer.styl

```diff
@@ -4,6 +4,9 @@ footer
   background-color: black
   background: #272727
   color: hsla(0,0%,100%,.5)
+  position absolute
```

Comment:
yeah tbh i just did this so that i could stop being annoyed at the footer being in the wrong place, we'll want to make an actual styling pass